### ISSUE TITLE
[question, MacOS] Is it possible to not quit app when user clicks X in the corner?

### ISSUE THREAD

**Issue by cvl** [nodegui/react-nodegui#193]
I.e. instead of closing app, it should close the window (kinda default behavior of most macos apps).

**Comment by cvl** [comment]
So far found that 
const qApp = QApplication.instance()
qApp.setQuitOnLastWindowClosed(false)

will not quit the app, but when I click the icon, the window doesn't appear again.

**Comment by master-atul** [comment]
Yeah, I think this would need some work at the moment since Qt doesnt really natively support this.
